metadata-ingestion/src/datahub/ingestion/source/usage/bigquery_usage.py

```diff
@@ -392,6 +422,22 @@ class BigQueryUsageConfig(DatasetSourceConfigBase, BaseUsageConfig):
     log_page_size: Optional[pydantic.PositiveInt] = 1000
     query_log_delay: Optional[pydantic.PositiveInt] = None
     max_query_duration: timedelta = timedelta(minutes=15)
+    credential: Optional[BigQueryCredential]
+    credentials_path: Optional[str] = None
+
+    @pydantic.root_validator()
+    def validate_config(cls, values: Dict[str, Any]) -> Dict[str, Any]:
+
+        if values["credential"]:
+            values["credentials_path"] = values[
```

Comment:
The `credential_path` is always being set if `credential` is set. Is this not supposed to be exposed to the users?